### PR TITLE
Add 422/444 in header file

### PR DESCRIPTION
**Pull request by lijing0010**
Signed-off-by: Jing Li <jing.b.li@intel.com>

### CODE REVIEW

File: Source/API/EbApi.h [hunk 1]
```diff
@@ -92,6 +92,13 @@ extern "C" {
 
 #define EB_BUFFERFLAG_EOS 0x00000001
 
+typedef enum EB_COLOR_FORMAT {
+    EB_YUV400,
+    EB_YUV420,
+    EB_YUV422,
+    EB_YUV444
+} EB_COLOR_FORMAT;
```
Comment (tianjunwork) [inline Source/API/EbApi.h]: better name with EB_CHROMAFORMAT_YUVXXX? 
typedef enum EB_COLOR_FORMAT  {
   EB_CHROMAFORMAT_YUV400 = 0,
  EB_CHROMAFORMAT_YUV_420        = 1,
  EB_CHROMAFORMAT_YUV_422        = 2,
  EB_CHROMAFORMAT_YUV_444        = 3,
  } EB_COLOR_FORMAT;

File: Source/API/EbApi.h [hunk 2]
```diff
@@ -138,6 +145,7 @@ typedef struct EB_H265_ENC_CONFIGURATION
     int32_t                 frameRateNumerator;
     int32_t                 frameRateDenominator;
     uint32_t                encoderBitDepth;
+    EB_COLOR_FORMAT         colorFormat;
```
Comment (tianjunwork) [inline Source/API/EbApi.h]: where is it used?

File: Source/API/EbApi.h (hunk 1 above)
Comment (lijing0010) [reply Source/API/EbApi.h]: it's not only about chroma, '4' denotes for luma actually

File: Source/API/EbApi.h (hunk 2 above)
Comment (lijing0010) [reply Source/API/EbApi.h]: I'm enabling 422/444 support, so it will be used for following patches

Comment (tianjunwork) [reply Source/API/EbApi.h]: where are the following patches? Since it's related, the PR can still be used to add the feature.

**Pull request closed without merging**